.circleci/config.yml

```diff
@@ -100,6 +100,14 @@ jobs:
           key: v1-jackdaw-repo-{{ .Branch }}-{{ .Revision }}
           paths:
             - .
+  lint:
```

Comment:
Could be slightly simpler:

https://github.com/FundingCircle/decisioning/blob/master/.drone.yml#L136